package.json

```diff
@@ -27,7 +27,7 @@
     "sbc-common-components": "2.4.8",
     "uuid": "^8.3.1",
     "v-mask": "^2.1.0",
-    "vue": "^2.6.11",
+    "vue": "2.6.12",
```

Comment:
I was getting errors and warning that vue and vue-template-compiler were out of sync. This is the only way to fix this atm since there is not a latest, matching version of both.